### ISSUE TITLE
[ENHANCEMENT] Image load is slow

### ISSUE THREAD

**Issue by YuriDevAT** [YuriDevAT/accessibility-first#45]
### Description

Due to the high resolution of the image for the thumbnail the page load is slow.

**Comment by pragyamishra56** [comment]
Mam, If it is possible then could you assign me this issue?

**Comment by YuriDevAT** [comment]
Thanks for your interest. This is not an open source project, therefore only I and my team are going to work on these issues.

**Comment by pragyamishra56** [comment]
If it is possible, may I join your team, ma'am? I would like to contribute my frontend skills.

**Comment by YuriDevAT** [comment]
No, unfortunately not. That's what open source projects are for, and organizations, where issues are properl created, PRs are guided and such.

**Comment by pragyamishra56** [comment]
Thank you for your response Mam
